python/tvm/relay/testing/mobilenet.py

```diff
@@ -0,0 +1,142 @@
+    act2 = relay.nn.relu(data=bn2)
+    return act2
+
+
+def mobile_net(num_classes=1000, image_shape=(1, 3, 224, 224),
+               dtype='float32', alpha=1.0, is_shallow=False):
+    """Function to construct a MobileNet"""
+    data = relay.var("data", shape=image_shape, dtype=dtype)
+    body = conv_block(data, 'conv_block_1', int(32*alpha), strides=(2, 2))
+    body = separable_conv_block(body, 'separable_conv_block_1',
+                                int(32*alpha), int(64*alpha))
+    body = separable_conv_block(body, 'separable_conv_block_2',
+                                int(64*alpha), int(128*alpha), downsample=True)
+    body = separable_conv_block(body, 'separable_conv_block_3',
+                                int(128*alpha), int(128*alpha))
+    body = separable_conv_block(body, 'separable_conv_block_4',
+                                int(128*alpha), int(256*alpha), downsample=True)
+    body = separable_conv_block(body, 'separable_conv_block_5',
+                                int(256*alpha), int(256*alpha))
+    body = separable_conv_block(body, 'separable_conv_block_6',
+                                int(256*alpha), int(512*alpha), downsample=True)
+    if is_shallow:
+        body = separable_conv_block(body, 'separable_conv_block_7',
+                                    int(512*alpha), int(1024*alpha), downsample=True)
+        body = separable_conv_block(body, 'separable_conv_block_8',
+                                    int(1024*alpha), int(1024*alpha), downsample=True)
+    else:
+        for i in range(7, 12):
+            body = separable_conv_block(body, 'separable_conv_block_%d' % i,
+                                        int(512*alpha), int(512*alpha))
+        body = separable_conv_block(body, 'separable_conv_block_12',
+                                    int(512*alpha), int(1024*alpha), downsample=True)
+        body = separable_conv_block(body, 'separable_conv_block_13',
+                                    int(1024*alpha), int(1024*alpha))
+    pool = relay.nn.global_avg_pool2d(data=body)
+    flatten = relay.nn.batch_flatten(data=pool)
+    weight = relay.var('fc_weight')
+    fc = relay.nn.dense(data=flatten, weight=weight, units=num_classes)
+    softmax = relay.nn.softmax(data=fc)
+    return relay.Function(relay.ir_pass.free_vars(softmax), softmax)
+
+
+def get_workload(batch_size=1, num_classes=1000, image_shape=(3, 224, 224), dtype='float32'):
+    """Get benchmark workload for mobilenet
+
+    Parameters
+    ----------
+    batch_size : int
+        The batch size used in the model
+
+    num_classes : int, optional
+        Number of classes
+
+    image_shape : tuple, optional
+        The input image shape
+
+    dtype : str, optional
+        The data type
+
+    Returns
+    -------
+    net : relay.Function
+        The computational graph
+
+    params : dict of str to NDArray
+        The parameters.
+    """
+    image_shape = tuple([batch_size] + list(image_shape))
```

Comment:
Should this be moved inside `mobile_net`? `resnet` takes `batch_size` as an argument.

Reply:
@eqy +1, maybe we need to be consistent, either pass batch_size as a separate argument, or rename to data_shape